types/persisted-model.d.ts

```diff
@@ -38,8 +38,14 @@ export declare class PersistedModel extends ModelBase {
   static create(
     data: PersistedData,
     options?: Options,
-    callback?: Callback<PersistedData>,
-  ): PromiseOrVoid<PersistedData>;
+    callback?: Callback<PersistedModel>,
+  ): PromiseOrVoid<PersistedModel>;
+
+  static create(
+    data: PersistedData[],
```

Comment:
Add an overload for batch create mode, where an array of data objects is accepted and the method returns an array of model instances created.